.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,6 +38,14 @@ https://github.com/aws/amazon-ecs-agent/commit/c9aefebc2b3007f09468f651f6308136b
 **Does this PR include breaking model changes? If so, Have you added transformation functions?**
 <!-- If yes, next release should have a upgraded minor version -->  
 
+**Additional Information**
```

Comment:
how about we make this a title and move it above the breaking model changes question above? and also add permalinks for the ecs-log-collector URLs.


like: 
```

### Additional information

**Does this PR include breaking model changes? If so, Have you added transformation functions?**
<!-- If yes, next release should have a upgraded minor version -->  

**Does this PR include the addition of new environment variables in the README?**
<!-- 
If it is a sensitive variable, add it to this block-list in ecs-logs-collector [here](https://github.com/aws/amazon-ecs-logs-collector/blob/b0958c2aa424c6dc578d5a8def4422c51791a076/ecs-logs-collector.sh#L63)

If it is not a sensitive variable, add it to the allow-list in ecs-logs-collector [here](https://github.com/aws/amazon-ecs-logs-collector/blob/b0958c2aa424c6dc578d5a8def4422c51791a076/ecs-logs-collector.sh#L66)
-->

```

Reply:
Yep, that makes a bit more sense actually. But I think permalinks don't work since those comments inside <!-- --> only appear when editing the PR comment, I'm not sure what to do about that. I'll try it out though

Reply:
you could try creating a new PR in your fork repo, with the PR template from this PR 😄

Reply:
It's as I suspected, there's no formatting so I think I'll keep the link directly in there instead of the formatting

<img width="889" alt="Screenshot 2024-08-07 at 11 17 49 AM" src="https://github.com/user-attachments/assets/8aee2565-48ae-4545-94c9-d60be02da47b">